guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -35,6 +35,21 @@ include::modules/proc_synchronizing-the-system-clock-with-chronyd.adoc[leveloffs
 ifdef::katello,satellite[]
 [id="configuring-capsule-server-with-ssl-certificates"]
 == Configuring {SmartProxyServer} with SSL Certificates
+
+
+ifdef::foreman-deb[]
+This procedure is only for Katello plug-in users.
+endif::[]
```

Comment:
I think that this sentence is not necessary in this case, because the whole section will not be generated for foreman-deb sice it's nested under the condition to be output only for katello and satellite.